### PR TITLE
Fix launch oracle client

### PR DESCRIPTION
**Pull request by yrong**

### CODE REVIEW

**Comment by yrong** [comment]
require companion with https://github.com/parallel-finance/oracle-client/pull/55

File: scripts/oracle/config/hko.json [hunk 1]
```diff
@@ -0,0 +1,334 @@
+{
+    "assets": [
```
Comment (GopherJ) [inline scripts/oracle/config/hko.json:2]: hard to synchronize this file

Comment (GopherJ) [reply scripts/oracle/config/hko.json:2]: can you move this file to: https://github.com/parallel-finance/apps-config under oracle directory, same change can be needed on oracle-client side

File: .gitmodules [hunk 2]
```diff
@@ -4,3 +4,6 @@
 [submodule "scripts/polkadot-launch"]
 	path = scripts/polkadot-launch
 	url = https://github.com/paritytech/polkadot-launch
+[submodule "scripts/apps-config"]
```
Comment (GopherJ) [inline .gitmodules:7]: can you move it to resources/ which makes more sense

**Comment by GopherJ** [comment]
```
e::OverweightServiced::OverweightIndex", "LiquidStaking::ClaimedFor::T::AccountId", "PolkadotXcm::NotifyTargetMigrationFail::VersionedMultiLocation", "AMM::LiquidityAdded::T::AccountId", "LiquidStaking::BondingExtra::DerivativeIndex", "Treasury::Deposit::BalanceOf<T, I>", "XcmHelper::XcmWeightFeeUpdated::XcmWeightFeeMisc<Weight, BalanceOf<T>>", "PolkadotXcm::NotifyOverweight::QueryId", "TechnicalCommittee::Closed::T::Hash", "Farming::PoolAdded::AssetIdOf<T>", "Democracy::Undelegated::T::AccountId", "Democracy::Seconded::T::AccountId", "PolkadotXcm::SupportedVersionChanged::MultiLocation", "Assets::Transferred::T::Balance", "Bridge::BridgeTokenFeeUpdated::BalanceOf<T>", "ParachainSystem::DownwardMessagesProcessed::relay_chain::Hash", "Proxy::Announced::CallHashOf<T>", "Farming::PoolCoolDownDurationChanged::AssetIdOf<T>", "LiquidStaking::Bonding::BalanceOf<T>", "LiquidStaking::Unstaked::BalanceOf<T>", "CumulusXcm::ExecutedDownward::[u8; 8]", "Multisig::MultisigCancelled::Timepoint<T::BlockNumber>", "Loans::Borrowed::T::AccountId", "DmpQueue::ExecutedDownward::Outcome", "System::KilledAccount::T::AccountId", "Democracy::Vetoed::T::BlockNumber", "Identity::IdentityKilled::BalanceOf<T>", "Loans::Redeemed::T::AccountId", "Loans::RepaidBorrow::AssetIdOf<T>", "CumulusXcm::ExecutedDownward::Outcome", "Crowdloans::VaultDoContributing::BalanceOf<T>", "Balances::Withdraw::T::Balance", "PolkadotXcm::Notified::QueryId", "PolkadotXcm::SupportedVersionChanged::XcmVersion", "Crowdloans::VaultContributed::T::AccountId", "Democracy::PreimageUsed::T::Hash", "Crowdloans::VaultCreated::VaultId", "Farming::AssetsRedeem::T::BlockNumber", "AssetRegistry::AssetTypeUpdated::T::AssetType", "AMM::Traded::BalanceOf<T, I>", "Identity::JudgementRequested::T::AccountId", "XcmpQueue::UpwardMessageSent::T::Hash", "Proxy::AnonymousCreated::T::AccountId", "PolkadotXcm::UnexpectedResponse::MultiLocation", "Loans::RepaidBorrow::T::AccountId", "Crowdloans::VaultRedeemed::VaultPhase", "DmpQueue::OverweightEnqueued::Weight", "Democracy::PreimageUsed::T::AccountId", "Bridge::ChainRegistered::ChainId", "Democracy::Delegated::T::AccountId", "Democracy::Tabled::T::AccountId", "Crowdloans::VaultClaimed::VaultPhase", "Farming::PoolAdded::T::BlockNumber", "PolkadotXcm::ResponseReady::Response", "LiquidStaking::Staked::T::AccountId", "Assets::AssetStatusChanged::T::AssetId", "Assets::Transferred::T::AssetId", "Crowdloans::VaultContributed::VaultId", "Farming::PoolUnlockHeightReset::AssetIdOf<T>", "Multisig::MultisigCancelled::T::AccountId", "ParachainSystem::ValidationFunctionApplied::RelayChainBlockNumber", "System::Remarked::T::Hash", "Farming::RewardPaid::T::AccountId", "Crowdloans::VaultDoContributing::VaultId", "Bridge::BridgeTokenFeeUpdated::CurrencyId", "Crowdloans::PartiallyMigrated::ParaId", "Crowdloans::AllRefunded::VaultId", "Crowdloans::VaultDoWithdrawing::VaultPhase", "Bridge::MaterializeInitialized::ChainNonce", "Bridge::MaterializeVoteAgainst::ChainNonce", "Prices::SetPrice::Price", "Loans::ReservesReduced::T::AccountId", "XcmpQueue::OverweightEnqueued::OverweightIndex", "Identity::JudgementUnrequested::T::AccountId", "Assets::Created::T::AssetId", "LiquidStaking::NotificationReceived::QueryId", "XcmpQueue::OverweightServiced::Weight", "Farming::AssetsWithdrew::T::BlockNumber", "AMM::PoolCreated::AssetIdOf<T, I>", "Bridge::BridgeTokenAccumulatedValueCleaned::BridgeType", "LiquidStaking::ExchangeRateUpdated::Rate", "Democracy::PreimageReaped::T::Hash", "Crowdloans::VaultCreated::BlockNumberFor<T>", "Bridge::MaterializeInitialized::ChainId", "Identity::SubIdentityAdded::BalanceOf<T>", "LiquidStaking::BondingExtra::BalanceOf<T>", "Assets::Issued::T::AssetId", "LiquidStaking::NotificationReceived::Box<MultiLocation>", "Assets::Burned::T::Balance", "Bridge::ProposalApproved::ChainNonce", "PolkadotXcm::NotifyTargetMigrationFail::QueryId", "Crowdloans::VaultCreated::ParaId", "XTokens::TransferredMultiAssets::T::AccountId", "Multisig::MultisigExecuted::T::AccountId", "Crowdloans::VaultWithdrew::VaultPhase", "AMMRoute::Traded::BalanceOf<T, I>", "Preimage::Requested::T::Hash", "XcmpQueue::Fail::XcmError", "Bridge::MaterializeVoteAgainst::T::AccountId", "Bridge::TeleportBurned::CurrencyId", "Scheduler::Scheduled::T::BlockNumber", "Crowdloans::NotificationReceived::Box<MultiLocation>", "Crowdloans::AllMigrated::VaultId", "Crowdloans::VaultDoContributing::ParaId", "CumulusXcm::InvalidFormat::[u8; 8]", "LiquidStaking::Unbonding::BalanceOf<T>", "CollatorSelection::CandidateRemoved::T::AccountId", "Loans::Borrowed::BalanceOf<T>", "Bridge::MaterializeInitialized::T::AccountId", "Bridge::BridgeTokenCapUpdated::BalanceOf<T>", "Prices::SetPrice::CurrencyId", "Assets::Created::T::AccountId", "XTokens::TransferredMultiAssets::MultiAssets", "DmpQueue::WeightExhausted::MessageId", "PolkadotXcm::AssetsTrapped::MultiLocation", "Treasury::Spending::BalanceOf<T, I>", "PolkadotXcm::VersionChangeNotified::XcmVersion", "Crowdloans::VaultDoWithdrawing::ParaId", "Assets::ApprovalCancelled::T::AccountId", "Crowdloans::VaultDissolved::ParaId", "Proxy::AnonymousCreated::T::ProxyType", "Vesting::VestingScheduleAdded::T::AccountId", "Loans::RewardWithdrawn::BalanceOf<T>", "Assets::Issued::T::Balance", "XcmpQueue::OverweightEnqueued::RelayBlockNumber", "Scheduler::CallLookupFailed::TaskAddress<T::BlockNumber>", "Balances::Slashed::T::Balance", "Loans::ReservesAdded::AssetIdOf<T>", "AssetRegistry::AssetRegistered::T::AssetId", "XTokens::TransferredMultiAssets::MultiLocation", "Crowdloans::VaultCreated::ContributionStrategy", "Proxy::ProxyAdded::T::AccountId", "Loans::DistributedBorrowerReward::BalanceOf<T>", "Scheduler::Canceled::T::BlockNumber", "Farming::PoolStatusChanged::AssetIdOf<T>", "Bridge::MaterializeInitialized::BalanceOf<T>", "AssetRegistry::AssetRegistered::T::AssetType", "PolkadotXcm::ResponseReady::QueryId", "DmpQueue::OverweightServiced::Weight", "XcmpQueue::BadFormat::T::Hash", "Proxy::ProxyAdded::T::ProxyType", "CollatorSelection::NewCandidacyBond::BalanceOf<T>", "Bridge::MaterializeMinted::ChainId", "Loans::MarketRewardSpeedUpdated::BalanceOf<T>", "GeneralCouncil::MemberExecuted::T::Hash", "Democracy::PreimageUsed::BalanceOf<T>", "Oracle::NewFeedData::T::AccountId", "LiquidStaking::Nominating::T::AccountId", "LiquidStaking::Unbonding::DerivativeIndex", "Identity::SubIdentityAdded::T::AccountId", "Bridge::ProposalRejected::ChainId", "Crowdloans::VaultRedeemed::T::AccountId", "Bridge::TeleportBurned::T::AccountId", "LiquidStaking::ReserveFactorUpdated::Ratio", "DmpQueue::WeightExhausted::Weight", "GeneralCouncil::Voted::T::Hash", "Loans::Deposited::AssetIdOf<T>", "Loans::CollateralAssetAdded::AssetIdOf<T>", "Crowdloans::NotificationReceived::QueryId", "LiquidStaking::Bonding::DerivativeIndex", "PolkadotXcm::Attempted::xcm::latest::Outcome", "Assets::ApprovedTransfer::T::AssetId", "Treasury::Rollover::BalanceOf<T, I>", "Assets::ApprovedTransfer::T::Balance", "Loans::RewardPaid::T::AccountId", "Loans::RewardWithdrawn::T::AccountId", "Treasury::Rejected::BalanceOf<T, I>", "AMM::Traded::T::AccountId", "Bridge::ChainRemoved::ChainId", "Bridge::TeleportBurned::ChainId", "PolkadotXcm::InvalidResponder::MultiLocation", "PolkadotXcm::NotifyTargetSendFail::MultiLocation", "Bridge::MaterializeVoteAgainst::ChainId", "LiquidStaking::StakingLedgerCapUpdated::BalanceOf<T>", "GeneralCouncil::Disapproved::T::Hash", "LiquidStaking::Rebonding::DerivativeIndex", "Loans::LiquidatedBorrow::AssetIdOf<T>", "PolkadotXcm::InvalidResponder::QueryId", "Crowdloans::VaultDissolved::VaultId", "Bridge::TeleportBurned::TeleAccount", "PolkadotXcm::NotifyTargetSendFail::QueryId", "Bridge::MaterializeMinted::ChainNonce", "Bridge::BridgeTokenStatusUpdated::CurrencyId", "AssetRegistry::AssetDeregisteredd::T::AssetType", "LiquidStaking::WithdrawingUnbonded::DerivativeIndex", "TechnicalCommittee::Disapproved::T::Hash", "Loans::DistributedSupplierReward::AssetIdOf<T>", "Assets::ApprovalCancelled::T::AssetId", "Democracy::Vetoed::T::AccountId", "Loans::Deposited::T::AccountId", "GeneralCouncil::Executed::T::Hash", "Identity::JudgementGiven::T::AccountId", "Balances::BalanceSet::T::AccountId", "Balances::BalanceSet::T::Balance", "Vesting::Claimed::T::AccountId", "DmpQueue::InvalidFormat::MessageId", "Treasury::Awarded::T::AccountId", "Balances::Slashed::T::AccountId", "Farming::RewardPaid::T::BlockNumber", "Multisig::NewMultisig::T::AccountId", "Crowdloans::VaultUpdated::BlockNumberFor<T>", "Loans::RepaidBorrow::BalanceOf<T>", "Farming::AssetsRedeem::T::AccountId", "Assets::MetadataCleared::T::AssetId", "AssetRegistry::UnitsPerSecondUpdated::T::AssetType", "LiquidStaking::NotificationReceived::XcmError", "Loans::ReservesReduced::BalanceOf<T>", "Preimage::Cleared::T::Hash", "Democracy::Tabled::BalanceOf<T>", "Loans::UpdatedMarket::Market<BalanceOf<T>>", "Farming::AssetsRedeem::BalanceOf<T>", "Identity::SubIdentityRemoved::T::AccountId", "Crowdloans::VaultDoContributing::T::AccountId", "Assets::OwnerChanged::T::AccountId", "Assets::Burned::T::AssetId", "GeneralCouncil::Proposed::T::Hash", "Crowdloans::VaultClaimed::BalanceOf<T>", "Bridge::MaterializeVoteFor::ChainId", "ParachainSystem::UpgradeAuthorized::T::Hash", "AMM::LiquidityRemoved::BalanceOf<T, I>", "Bridge::BridgeTokenRegistered::CurrencyId", "Sudo::KeyChanged::T::AccountId", "Crowdloans::VaultRedeemed::AssetIdOf<T>", "Crowdloans::PartiallyMigrated::VaultId", "Assets::Thawed::T::AssetId", "PolkadotXcm::NotifyTargetSendFail::XcmError", "Treasury::Burnt::BalanceOf<T, I>", "Farming::AssetsRedeem::AssetIdOf<T>", "Farming::PoolUnlockHeightReset::T::BlockNumber", "Loans::RewardAdded::T::AccountId", "Bridge::TeleportBurned::ChainNonce", "Balances::Endowed::T::Balance", "Crowdloans::VaultCreated::TrieIndex", "AMMRoute::Traded::T::AccountId", "DmpQueue::UnsupportedVersion::MessageId", "CollatorSelection::NewInvulnerables::T::AccountId", "Farming::AssetsWithdrew::AssetIdOf<T>", "Democracy::PreimageInvalid::T::Hash", "Crowdloans::VaultUpdated::ContributionStrategy", "Bridge::BridgeTokenRegistered::BalanceOf<T>", "Crowdloans::VaultContributed::BalanceOf<T>", "Democracy::PreimageNoted::T::Hash", "Loans::CollateralAssetRemoved::T::AccountId", "Balances::Reserved::T::AccountId", "Loans::ReservesAdded::T::AccountId", "Democracy::PreimageMissing::T::Hash", "Bridge::MaterializeVoteAgainst::CurrencyId", "Streaming::StreamCreated::BalanceOf<T>", "Identity::IdentitySet::T::AccountId", "Assets::Destroyed::T::AssetId", "Bridge::ProposalApproved::ChainId", "System::NewAccount::T::AccountId", "PolkadotXcm::VersionChangeNotified::MultiLocation", "Crowdloans::VaultUpdated::BalanceOf<T>", "Farming::AssetsWithdrew::BalanceOf<T>", "Assets::TransferredApproved::T::Balance", "TechnicalCommittee::Voted::T::Hash", "TechnicalCommittee::Voted::T::AccountId", "Bridge::MaterializeMinted::CurrencyId", "Balances::ReserveRepatriated::T::AccountId", "Farming::PoolCoolDownDurationChanged::T::BlockNumber", "DmpQueue::OverweightEnqueued::OverweightIndex", "Identity::SubIdentityRevoked::BalanceOf<T>", "AMM::LiquidityRemoved::AssetIdOf<T, I>", "Crowdloans::VaultPhaseUpdated::ParaId", "Streaming::StreamWithdrawn::StreamId", "XcmpQueue::OverweightEnqueued::ParaId", "Loans::Redeemed::BalanceOf<T>", "Loans::RewardAdded::BalanceOf<T>", "Bridge::TeleportBurned::BalanceOf<T>", "Crowdloans::VaultWithdrew::BalanceOf<T>", "DmpQueue::OverweightEnqueued::MessageId", "DmpQueue::ExecutedDownward::MessageId", "PolkadotXcm::InvalidResponderVersion::QueryId", "Balances::ReserveRepatriated::T::Balance", "AMM::PoolCreated::T::AccountId", "LiquidStaking::Matching::BalanceOf<T>", "Balances::Unreserved::T::AccountId", "Assets::Issued::T::AccountId", "Prices::ResetPrice::CurrencyId", "Farming::RewardAdded::BalanceOf<T>", "Assets::ForceCreated::T::AccountId", "TechnicalCommittee::Proposed::T::AccountId", "Loans::MarketRewardSpeedUpdated::AssetIdOf<T>", "AMM::LiquidityAdded::BalanceOf<T, I>", "PolkadotXcm::NotifyDispatchError::QueryId", "GeneralCouncil::Proposed::T::AccountId", "Crowdloans::VaultDoWithdrawing::VaultId", "Crowdloans::AllRefunded::ParaId", "PolkadotXcm::NotifyDecodeFailed::QueryId", "Multisig::MultisigApproval::T::AccountId", "Balances::DustLost::T::AccountId", "Crowdloans::VaultClaimed::ParaId", "Streaming::StreamCanceled::StreamId", "Farming::PoolStatusChanged::T::BlockNumber", "PolkadotXcm::NotifyOverweight::Weight", "GeneralCouncil::Voted::T::AccountId", "Crowdloans::VaultUpdated::VaultId", "Assets::ApprovedTransfer::T::AccountId", "Proxy::ProxyAdded::T::BlockNumber", "Bridge::BridgeTokenCapUpdated::CurrencyId", "Streaming::StreamCreated::StreamId", "LiquidStaking::ClaimedFor::BalanceOf<T>", "Assets::AssetThawed::T::AssetId", "Crowdloans::PartiallyRefunded::VaultId", "Balances::Transfer::T::Balance", "AssetRegistry::AssetDeregisteredd::T::AssetId", "PolkadotXcm::UnexpectedResponse::QueryId", "Preimage::Noted::T::Hash", "LiquidStaking::Rebonding::BalanceOf<T>", "OrmlXcm::Sent::MultiLocation", "Loans::RewardPaid::BalanceOf<T>", "Assets::Transferred::T::AccountId", "Crowdloans::VaultUpdated::ParaId", "Scheduler::Dispatched::TaskAddress<T::BlockNumber>", "Bridge::BridgeTokenRemoved::AssetIdOf<T>", "Loans::LiquidatedBorrow::T::AccountId"} If any events containing these types are received, this can cause a `TypeSizeUnavailable` error and prevent decoding the actual event being listened for.Use `ClientBuilder::register_type_size` to register missing type sizes.
```
do we have type size issues

**Pull request merged**